djangocms_versioning/models.py

```diff
@@ -95,6 +95,19 @@ class Meta:
     def __str__(self):
         return "Version #{}".format(self.pk)
 
+    def delete(self, using=None, keep_parents=False):
+        ContentModel = self.content._meta.model
+        grouper_name = self.grouper._meta.model_name
+        querydict = {'{}__pk'.format(grouper_name): self.grouper.pk}
+        count = ContentModel._original_manager.filter(**querydict).count()
+
+        grouper = self.grouper
```

Comment:
Is the assignment here redundant when it's only used once: grouper.delete()

Reply:
It is not redundant, when we execute the `delete` function `self` no longer contains grouper. Test fails if you change this, so there is a safety net for this.